app/serializers/resource_serializer.rb

```diff
@@ -2,8 +2,9 @@
 
 class ResourceSerializer < ActiveModel::Serializer
   type "resource"
-  attributes :id, :name, :abbreviation, :description, :onesky_project_id, :total_views, :manifest
-  attribute :resource_type_name, key: "resource-type"
+  attributes :id, :name, :abbreviation, :description, :onesky_project_id, :manifest
+  attribute :resource_type_name, key: :"resource-type"
+  attribute :total_views, key: :"total-views"
```

Comment:
I'm really surprised this works since I've never seen a symbol with quotes like that, but it does work so that's great.

Reply:
Nope, you can leave it. It's new to me, so I'm glad that I know it works. I might have to play around with it to see how it works in other situations.